views/users/landing.html

```diff
@@ -6,34 +6,36 @@
 
 {% block header %}
   {{ super() }}
-  <h1>{{title}}</h1>
 {% endblock %}
 
 {% block main %}
+<h1>{{title}}</h1>
 <p><a href="{{config.base}}/{{config.actions.add}}">{{config.base}} | {{config.actions.add}}</a></p>
 <h2>Content</h2>
 
-{% if users %}
-  <table>
-    <tr><th>email</th><th>role</th><th>{{config.actions.edit}}</th><th>{{config.actions.delete}}</th><th>date</th></tr>
-  {% for u in users %}
-    {% if u.email %}
-      <tr>
-        <td>{{u.email}}</td>
-        <td>{{u.role}}</td>
-        <td><a href="{{config.base}}/{{u.id}}/{{config.actions.edit}}">{{config.actions.edit}}</a></td>
-        <td>
-        {% if u.id != user.id %}
-        <a href="{{config.base}}/{{u.id}}/{{config.actions.delete}}">{{config.actions.delete}}</a>
-        {% endif %}
-        </td>
-        <td>{{u.created}}</td>
-      </tr>
+  <div class="table-fluid">
```

Comment:
@scottnath ahhh yeah, good catch. I didn't update the content one yet, because it wasn't working with pulling stuff in last time I went in and styled. I'll wait on Sams feed back on whether or not to use tables or divs.